Dockerfile-ultra-lite

```diff
@@ -8,39 +8,39 @@ ENV DOCKER_ENABLE_SECURITY=false \
     HOME=/home/stirlingpdfuser \
     VERSION_TAG=$VERSION_TAG \
     JAVA_TOOL_OPTIONS="$JAVA_TOOL_OPTIONS -XX:MaxRAMPercentage=75"
-#	PUID=1000 \
+#   PUID=1000 \
 #   PGID=1000 \
 #   UMASK=022 \
 
+# Copy necessary files
+COPY scripts/download-security-jar.sh /scripts/download-security-jar.sh
+COPY scripts/init-without-ocr.sh /scripts/init-without-ocr.sh
+COPY pipeline /pipeline
+COPY build/libs/*.jar app.jar
+
 # Create user and group using Alpine's addgroup and adduser
 #RUN addgroup -g $PGID stirlingpdfgroup && \
 #    adduser -u $PUID -G stirlingpdfgroup -s /bin/sh -D stirlingpdfuser && \
```

Comment:
is there a reason why this is disabled? Can it be removed? Or should I try to get it working in PR after this is merged?

Reply:
Previously i had some issues with it and not had time to resolve
I would love to have correct permissions and user applied